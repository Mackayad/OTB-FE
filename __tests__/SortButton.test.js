import { shallow } from 'enzyme';
import SortButton from "../components/sort/sort-button"

describe("Sort button component", () => {

  let wrapper

  beforeEach(() => {
    wrapper = shallow(<SortButton key={0} onButtonClick={ jest.fn() } id={0} isActive={ false } >Test Button</SortButton>)
  });

  it('Should render correctly', () => expect(wrapper).toMatchSnapshot())

  it(`Should have text equal to provided value 'Test Button'`, () => {
    expect(wrapper.text()).toEqual('Test Button')
  })

})