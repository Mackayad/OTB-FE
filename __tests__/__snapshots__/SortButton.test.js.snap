// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sort button component Should render correctly 1`] = `ShallowWrapper {}`;
